MIDIchlorians/MIDIchlorians/DataManager.swift

```diff
@@ -92,16 +96,20 @@ class DataManager {
         }
 
         do {
-            guard let session = loadSession(sessionName) else {
+            guard let session = loadExactSession(sessionName) else {
```

Comment:
what are we doing here? looks like we saved the session, then we are loading it again?